Sources/xcbeautify/Xcbeautify+Help.swift

```diff
@@ -0,0 +1,13 @@
+import ArgumentParser
+
+extension Xcbeautify {
+	private static var abstract: String = "Format the output of `xcodebuild` and `swift`"
+	private static var additionalHelp: String = """
+	EXAMPLE: xcodebuild test -project MyApp.xcodeproj -scheme MyApp -destination 'platform=iOS Simulator,OS=12.0,name=iPhone X' | xcbeautify
+	"""
+	static var configuration: CommandConfiguration = {
+		return CommandConfiguration(abstract: abstract,
+											 discussion: additionalHelp,
```

Comment:
I use Xcode with 3 space tabs, so this might be misaligned with different setups.

Reply:
Could you reformat all Swift files with 4 spaces indentation (which is the style this project is using)?